frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -250,11 +250,13 @@ pub mod helpers;
 const LOG_TARGET: &'static str = "runtime::election-provider";
 
 pub mod unsigned;
+pub mod signed;
 pub mod weights;
 
-/// The weight declaration of the pallet.
 pub use weights::WeightInfo;
 
+use signed::{SignedSubmission, BalanceOf, NegativeImbalanceOf, PositiveImbalanceOf};
```

Comment:
```suggestion
pub use signed::{SignedSubmission, BalanceOf, NegativeImbalanceOf, PositiveImbalanceOf};
```

since I will use this pallet quite a lot in the miner, I generally make almost everything public for now